app/.gitignore

```diff
@@ -8,11 +8,11 @@
 /web/libraries/
 
 # Ignore sensitive information
-# /web/sites/*/settings.php
+/web/sites/*/settings.php
 /web/sites/*/settings.local.php
 
 # Ignore Drupal's file directory
-# /web/sites/*/files/
+/web/sites/*/files/
```

Comment:
You might want to double-check with Ryan. I think he commented these out of the gitignore for a reason. I'm just not sure what it was :-)

Reply:
also, from looking at below, I'm wondering if /app/config/sync/* should be added. Do we need to be adding those files into the repo?